opentracing-api/src/main/java/io/opentracing/Span.java

```diff
@@ -111,4 +122,19 @@
      * @return this Span instance, for chaining
      */
     Span setOperationName(String operationName);
+
+    /**
+     * @deprecated use {@link #log(Map)} like this
+     * {@code span.log(Map.of("event", "timeout"))}
```

Comment:
We would like to define something akin to https://github.com/opentracing/opentracing-java/blob/master/opentracing-api/src/main/java/io/opentracing/tag/Tags.java for logging. I had hoped to get the basic KV mechanism in place first, then gather input about logging keys that ought to be standardized and promoted in this way.